src/AssetMapper/SassCssCompiler.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Symfonycasts\SassBundle\AssetMapper;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\AssetMapper\Compiler\AssetCompilerInterface;
+use Symfony\Component\AssetMapper\MappedAsset;
+
+class SassCssCompiler implements AssetCompilerInterface
+{
+    public function __construct(
+        private string $scssPath,
+        private string $cssPath
+    ) {}
+
+    public function supports(MappedAsset $asset): bool
+    {
+        return realpath($asset->sourcePath) === realpath($this->scssPath);
+    }
+
+    public function compile(string $content, MappedAsset $asset, AssetMapperInterface $assetMapper): string
+    {
+        $asset->addFileDependency($this->scssPath);
```

Comment:
Actually, this should be the final, output CSS file - so `$this->cssPath` I believe. What we're saying here is:

> Hey AssetMapper! When the final, built CSS file changes, you need to bust the cache on this asset and serve the fresh contents.